tests/lib/rules/camelcase.js

```diff
@@ -151,6 +151,11 @@ ruleTester.run("camelcase", rule, {
             code: "import { no_camelcased as camelCased, anoterCamelCased } from \"external-module\";",
             parserOptions: { ecmaVersion: 6, sourceType: "module" }
         },
+        {
```

Comment:
Can we add a test that also explicitly sets the config to `false`?

Reply:
When it's `false`, that is a "invalid" case.

Reply:
I understand that it’s the default value. I think it’s useful as a regression test though.

Reply:
That's "invalid" case, while here are "valid".

Reply:
Sorry, maybe I'm not being clear, I think it's valuable to have a regression test like:

```js
{
    code: 'import { notCamelcase } from "external module";'.
    options: [{ ignoreImports: false }],
    parserOptions: { ecmaVersion: 6, sourceType: "module" }
}
```

Reply:
`notCamelcase`?